README.md

```diff
@@ -36,6 +36,18 @@ Beta version of the ROS2 Universal Robots driver. Should be transferred to the U
 **NOTE**: If you are a ROS2 beginner and get lost consult the [references - section](#references).
 
 
+### How to Setup the UR Robot
```

Comment:
Please use 2X ## like the sections above and below

Reply:
For formatting suggestions I have found it least confusing to simply do "Insert a suggestion."